src/icons/icon.tsx

```diff
@@ -0,0 +1,27 @@
+import { type SVGProps } from 'react';
+import spriteHref from './sprite.svg';
+import type { IconName } from './icon-constant';
+
+const defaultIconSizes = {
+  small: 18,
+  large: 24,
+};
```

Comment:
let's use as const here

```suggestion
const defaultIconSizes = {
  small: 18,
  large: 24,
} as const;
```